BanchoSharp/Multiplayer/MultiplayerPlayer.cs

```diff
@@ -7,19 +7,24 @@ public enum TeamColor
 	None
 }
 
-// todo: Track player mods
-
 public class MultiplayerPlayer
 {
 	public string Name { get; }
 	public TeamColor Team { get; set; }
 	public int Slot { get; set; }
+	
+	// The mods the player is using, these only get updated after "!mp settings" has been ran.
+	public Mods Mods { get; set; }
+
+	public int? Score { get; set; } = null;
+	public bool? Passed { get; set; } = null;
```

Comment:
The default value of a nullable value type is null, so no need to declare that here.